.github/workflows/autoupdate-preview.yaml

```diff
@@ -1,13 +1,8 @@
-name: Generate Dev SDK 
+name: Generate Preview SDK 
 on:
   workflow_dispatch:
   schedule:
    - cron: '30 8 * * 1,3,5'
-  push:
```

Comment:
We no longer need that for the dev job as tools became much more than just generation folder this creates number of unneded PR updates .